Library/Homebrew/cmd/update.sh

```diff
@@ -219,8 +219,16 @@ pull() {
 
   if [[ -z "$HOMEBREW_MERGE" ]]
   then
+    if [[ -n "$HOMEBREW_VERBOSE" ]]
+    then
+      echo "Rebasing $DIR onto origin/$UPSTREAM_BRANCH ..."
+    fi
     git rebase "${QUIET_ARGS[@]}" "origin/$UPSTREAM_BRANCH"
   else
+    if [[ -n "$HOMEBREW_VERBOSE" ]]
+    then
+      echo "Merging origin/$UPSTREAM_BRANCH into $DIR..."
```

Comment:
Personally, in "ASCII art" I would not use "LaTeX quality" and add a space before `...` in this case, in order to avoid the misinterpretation you pointed out.